### PR TITLE
Docs dark theme rebased

### PR DESCRIPTION
**Pull request by kwiato**

### CODE REVIEW

**Comment by dgarcia360** [comment]
Some comments:

- Icon not rendering as expected:

image

- Icon not aligned:

image

- Icon significant smaller than the previous version -> is it intended?

image

- Review default label color in light mode:

image

- Review announcement button color in dark mode:

image

- Review icons spacing:

image


- The "User mailing list" icon should not render, we removed it recently:

image

- Dropdown arrows are smaller -> is it intended?

image

- Test do not pass. I'll send the fix.

File: src/css/components/_icons.scss
Comment (dgarcia360) [inline src/css/components/_icons.scss:7]: Looks safe to remove part of these icons if we do not use them anymore and they are defined in the new icons font.

Comment (kwiato) [reply src/css/components/_icons.scss:7]: I wanted to wait untill tested in other docs

File: src/css/base/variables/_colors.scss
Comment (dgarcia360) [inline src/css/base/variables/_colors.scss:1]: We are currently defining colors in two separate files: `base/_variables.scss` and `variables/_colors.scss`. To improve maintainability, we should consolidate these into a single file. Additionally, we should remove any unused variables and colors.

File: src/css/main.scss
Comment (dgarcia360) [inline src/css/main.scss:1]: Why is this change necessary?

File: src/css/base/variables/_colors.scss
Comment (kwiato) [reply src/css/base/variables/_colors.scss:1]: Agree, just want to make sure I don't break anything right now.

**Comment by dgarcia360** [comment]
**Feedback:**

- The "x" button to close the announcement banner has disappeared:

image

- When navigating between pages in dark mode, the light mode briefly flashes for a few milliseconds, causing a flash effect. We can work on this as part of a separate PR. To evaluate if it's a blocker for releasing.

- Review the footer in responsive mode:

    image

   Before:
   
    image

- It's important to clean up the codebase before the release:

   > We are currently defining colors in two separate files: base/_variables.scss and variables/_colors.scss. To improve maintainability, we should consolidate these into a single file. Additionally, we should remove any unused variables and colors.
   
   Styles are only defined in this project for all projects, so it should be safe to update them as long as we test them. If you'd prefer not to address this in the current PR, we can handle it as part of a separate one.

**Comment by dgarcia360** [comment]
Fixed tests: https://github.com/scylladb/sphinx-scylladb-theme/pull/1210/commits/70811e4f375a8d7d6d59ca156e3d7a01c24ef81b

**Comment by dgarcia360** [comment]
Icons that are not using the new icons library:

* Left sidebar expand button (`fas fa-bars`): 
    image
    
* Tables checks (`fa fa-check`) and cross marks (`fa fa-times`)
    image
    
* Hero box & topic documentation examples: `/examples/hero-box` & `/examples/topic-box`

**Comment by dgarcia360** [comment]
LGTM. Let's wait for @annastuchlik review before merging.

New issues derived from the changes in this PR:

- https://github.com/scylladb/sphinx-scylladb-theme/issues/1219
- https://github.com/scylladb/sphinx-scylladb-theme/issues/1218
- https://github.com/scylladb/sphinx-scylladb-theme/issues/1220
- https://github.com/scylladb/sphinx-scylladb-theme/issues/1221

To decide if we want to include them in the next release.

**Review by dgarcia360 — APPROVED** [review]

**Comment by tzach** [comment]
@dgarcia360 why cant I see dark node on the Theme site?  https://sphinx-theme.scylladb.com/stable/

**Pull request merged**
